test/mbta_v3_api/facilities/repo_test.exs

```diff
@@ -18,7 +18,7 @@ defmodule Facilities.RepoTest do
     long_name: "Alewife Escalator 349 (Platform to Russell Field)",
     stop: %MBTAV3API.Stops.Stop{
       id: "place-alfcl",
-      type: "stop"
+      name: "Alewife"
```

Comment:
Cannot immediately tell why this test changed?

Reply:
I changed the parsing of stops that come back with facilities a little bit- I removed the "type" because it's confusing and doesn't add new information (it always says "stop"), and I added the name if it's available because I need it in downstream work. Unfortunately `type` was removed in a previous PR but this test wasn't updated because I missed it and we don't have CI in here yet 😬 but the "name" parsing change can be seen in lib/mbta_v3_api/facilities/facility.ex. Basically if you add `include: stop` to the params, the stop comes back with an `attributes` field and we can get the name off of there.